pkg/computeclasses/computeclasses.go

```diff
@@ -37,7 +37,7 @@ func parseQuantity(memory string) (resource.Quantity, error) {
 	return resource.ParseQuantity(memory)
 }
 
-func ParseComputeClassMemory(memory internaladminv1.ComputeClassMemory) (memoryQuantities, error) {
+func ParseComputeClassMemoryAPI(memory apiv1.ComputeClassMemory) (memoryQuantities, error) {
```

Comment:
nit: I suggest we rename this function to `ParseComputeClassMemory`, and the other to `ParseComputeClassMemoryInternal`, as I think those names are a bit clearer.

Reply:
➕